mmf/models/mmbt.py

```diff
@@ -352,6 +352,43 @@ def forward(self, sample_list):
         if self.config.use_modal_end_token:
             modal_end_token = sample_list.input_ids[:, -1].clone().detach()
 
+        if hasattr(sample_list, "modal_token_type_ids"):
+            modal_token_type_ids = sample_list.modal_token_type_ids
+        else:
+            modal_token_type_ids = torch.zeros(
+                (input_modal.size(0)),
```

Comment:
Will the shape of the segment be always Bx1?

Reply:
We are relying on the broadcasting here.